account.go

```diff
@@ -0,0 +1,49 @@
+package quickbooks
+
+import (
+	"fmt"
+)
+
+type AccountQueryResponse struct {
+	Accounts []*Account `json:"Account"`
+}
+
+type Account struct {
+	ID      string `json:"Id"`
+	Name    string `json:"Name"`
+	Active  bool   `json:"Active"`
+	Type    string `json:"AccountType"`
+}
+
+type AccountFilter struct {
+	IsActive string // must be either "true" or "false"
+	Type     string
+}
+
+func (a *AccountFilter) Eq() string {
+	sql := "SELECT * FROM Account"
+	if a.IsActive == "true" || a.IsActive == "false" {
```

Comment:
If you're going to assume there's already a WHERE clause before then AND below, seems like you might as well just skip the conditional here.